src/builder/create_invite.rs

```diff
@@ -284,20 +281,19 @@ impl<'a> Builder for CreateInvite<'a> {
     async fn execute(
         self,
         cache_http: impl CacheHttp,
-        ctx: Self::Context<'_>,
+        channel: Self::Context<'_>,
```

Comment:
```suggestion
        ctx: Self::Context<'_>,
```
All other builders still name this param `ctx`.